server/bin/unittests

```diff
@@ -101,8 +106,9 @@ function _run_allscripts {
     _run pbench-copy-sosreports
     _run pbench-index
     _run pbench-index --tool-data
-    # These four are independent, running periodically to accomplish
+    # This next set are independent, running periodically to accomplish
```

Comment:
s/are/is ?

Reply:
Peter's phrase there is "The members of this next set are...", and "he members of t" has been removed for brevity.  😉

Reply:
The goal of the change was to keep the set of changes small since this is on `b0.69` branch.  As a team we seem to be willing to apply the same level of deep introspection of a change that we apply to the `main` branch (where we want to reduce the amount of unmaintainable code) to this legacy branch.

I'd like us as a team to reduce the amount of change we request on the legacy branches to a minimum.

Having said that ...